### PR TITLE
[python/en] Python decorators example added

### PR DESCRIPTION
**Pull request by asyne**
- [x] PR touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at _intermediate to experienced programmers_ (this is a poor format for explaining fundamental programming concepts)
- [x] YAML Frontmatter formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Seriously, look at it now. Watch for quotes and double-check field names.

### CODE REVIEW

File: python.html.markdown [hunk 1]
```diff
@@ -734,6 +735,26 @@ print(gen_to_list)  # => [-1, -2, -3, -4, -5]
 
 
 # Decorators
+# Decorator is a higher order function, which accepts and returns a function.
```
Comment (vendethiel) [inline python.html.markdown]: A decorator

File: python.html.markdown [hunk 2]
```diff
@@ -734,6 +735,26 @@ print(gen_to_list)  # => [-1, -2, -3, -4, -5]
...
+# In our case decorator accepts a function that is wrapped by decorator as a
```
Comment (vendethiel) [inline python.html.markdown]: I don't understand this sentence – is the first "decorator" supposed to be "add_apples"? Explaining what is a decorator with "decorator" is confusing :)

**Comment by asyne** [comment]
@vendethiel updated from upstream, requested changes are also included.

**Comment by vendethiel** [comment]
Thanks!

**Pull request merged**